<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportTimelapseDialog</class>
 <widget class="QDialog" name="ExportTimelapseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Rapid Review</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>16</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="initialLengthTitleLabel">
     <property name="text">
      <string>Initial Video Length</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLabel" name="resultLengthTitleLabel">
     <property name="text">
      <string>Exported Video Length</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="initialLengthLabel">
     <property name="text">
      <string notr="true">8h 37min</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="resultLengthSpinBox"/>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLabel" name="speedTitleLabel">
     <property name="text">
      <string>Rapid Review speed</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLabel" name="framesTitleLabel">
     <property name="text">
      <string>Frames Interval</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLabel" name="framesLabel">
     <property name="text">
      <string notr="true">1m 45s</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="resultLengthUnitsComboBox"/>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QSlider" name="speedSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="speedSpinBox">
     <property name="suffix">
      <string notr="true">x</string>
     </property>
     <property name="value">
      <number>52</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
